src/main/java/org/candlepin/controller/PoolManager.java

```diff
@@ -1541,38 +1548,33 @@ public void regenerateDirtyEntitlements(Iterable<Entitlement> entitlements) {
     }
 
     /**
-     * List entitlement pools.
+     * Lists available entitlement pools that match the provided criteria.
      *
-     * If a consumer is specified, a pass through the rules will be done for
-     * each potentially usable pool.
+     * @param qualifier
+     *  a qualifier with defined criteria for determining which pools should be returned
```

Comment:
can we add a javadoc link to the PoolQualifier method here? it makes it easier to click on it and read the docs on that one, now that the inputs are not explicitly listed on this method's javadoc